locales/en.yml

```diff
@@ -53,6 +54,22 @@ en:
                 --script-readable  display information in a script readable format
                 -h, --help         print this help
 
+        restart: |-
+          Restarts the given systemd service in the box
+
+          Usage: vagrant service-manager restart <object> [options]
+
+          Object:
+              Object can be any available service configured in the box using systemd unit files
+
```

Comment:
> docker, kubernetes (or k8s), or openshift.

Does this limits to only three services? Right now it looks like `sccli` or `systemd` services are only `docker, kubernetes or openshift`. 
IMO it would be better if we re-write it as `docker, kubernetes (or k8s), openshift etc`. This gives hint to user that you can user more `systemd` services too like `httpd` etc.

What do you think @bexelbie?